src/mono/wasm/runtime/cjs/dotnet.cjs.extpost.js

```diff
@@ -1,3 +1,4 @@
+var require = require || undefined;
```

Comment:
what does this do? please add a comment

Reply:
This captures require function from top level scope and also allows us to re-assign it to anything that user provided in `imports.require` field of `createDotnetRuntime`. I don't want to add comments here, because they are not stripped by the emcc compiler.